woocommerce/woocommerce
# Fix css date and time inputs display in order admin

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

The current 3.5em width of the time inputs obscures the values due to input spinner controls in certain browsers. Increasing to 4em resolves the issue. Also the width of the date input forces the minute input onto the line below. By changing the width to 40% the inputs are all aligned on a single row.

Example of before and after in Firefox:

![image](https://github.com/user-attachments/assets/6c9acdaa-e1d9-4514-853c-27d89be44ea1)

![image](https://github.com/user-attachments/assets/e7859798-dcf5-41f1-9eb4-9dc1c54aa5d1)

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Open the Edit Order page in Wordpress Admin
2. Look at the Date Created section of General at the top of the page in the Order Details section
3. The 3 inputs are on the line below, and should align in a single row with the changes. The spinner controls should no longer obscure the number on the edge, and have some white space between the last digit and the spinner controls.

### Changelog entry











-   [] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message Fix CSS for date and time inputs on edit order page in Wordpress admin

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment I was unable to run pnpm after installing with both homebrew and npm. I kept getting the following error, and couldn't find a resolution to it: ERROR  Failed to switch pnpm to v9.1.3.

</details>
